docs/stdlib/datetime.rst

```diff
@@ -101,35 +101,34 @@ Dates and Times
 
 .. _ref_std_datetime_intro:
 
-EdgeDB has two classes of date/time types:
+In EdgeDB, there are two types of date/time constructs:
```

Comment:
I think I prefer the original here since it's in active voice. I'd also lean toward "types" over "constructs" even though I believe both are accurate, but the constructs map to actual types which feels like the more relevant info in this context.